packages/server/src/types/accounts-server-options.ts

```diff
@@ -24,4 +25,6 @@ export interface AccountsServerOptions {
   siteUrl?: string;
   prepareMail?: PrepareMailFunction;
   sendMail?: SendMailType;
+  tokenCreator?: TokenCreator;
```

Comment:
Does it needs to be a separate interface or can it be just a function?

Reply:
as I think about our use-case, the creator should receive "static" options. so I think it will be better to implement it as an interface that you can instantiate ratter than passing the options in the function